### PR TITLE
Updated to rename manuscript category to "Simulation Analyses"

### PR DESCRIPTION
**Pull request by mrshirts**

### CODE REVIEW

File: _author_instructions/04_software_analyses.md [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
+permalink: /authors/software_analyses/
 ---
 
-## What is a publishable comparison for molecular simulation packages?
+## What is a publishable software analysis paper for molecular simulation packages?
 
-Simulation comparison papers describe attempts to perform the same
-calculation with a range of different simulation programs. 
+Software analysis papers describe attempts to perform the same
+calculation with a range of different simulation programs, perform anaysis of changes in perfomrmance over a series of releases of a single software package.  In some cases, comparisons of the same calculation with different methods may also be appropriate for a software analysis paper.
```
Comment (dmzuckerman) [inline _author_instructions/04_software_analyses.md]: "OR perform analysis of changes"
Do we want to state that such analyses are intended for packages with a significant publication record and user base?  Or is it ok to compare a new MD package to gromacs, for example?

File: _author_instructions/04_software_analyses.md [hunk 2]
```diff
@@ -1,28 +1,29 @@
...
+ 
 Such comparison should be updated periodically with different versions of
 the same programs (or potentially additional programs).
 
 
-## Additional criteria considered in review of comparisons of molecular simulation programs
+## Additional criteria considered in review of software analysis papers
 * Does the manuscript include developers or advanced users of all programs being compared to ensure that comparisons are fair?
-* Are best practices being used in the simulation comparisons?
+* Are best practices being used in the software analysis?
```
Comment (dmzuckerman) [inline _author_instructions/04_software_analyses.md]: What sort of best practices do we mean here?  Perhaps @jppiquem might like to weigh in.  For benchmarking?  Statistical uncertainty?  Let's be as specific as possible or refer to a specific set of standards.

File: _data/navigation.yml [hunk 3]
```diff
@@ -98,25 +96,27 @@ authors_perpetual_reviews.md:
     - title: Revision schedule for perpetual reviews
       url: /authors/perpetual_reviews/index.html#revision-schedule-for-perpetual-reviews
 
-authors_compare_simulations.md:
-  - title: Comparisons of Molecular Simulation Packages
-    url: /authors/compare_simulations/index.html#
+authors_best_practices.md:
```
Comment (dmzuckerman) [inline _data/navigation.yml]: Not sure if this is a mix-up: gihub shows compare_sim being changed to best_practices.
Likewise below

File: _author_instructions/04_software_analyses.md (hunk 2 above)
Comment (mrshirts) [reply _author_instructions/04_software_analyses.md]: I added some suggestions, would be good to get feedback from @jppiquem

File: _author_instructions/04_software_analyses.md (hunk 1 above)
Comment (mrshirts) [reply _author_instructions/04_software_analyses.md]: Added some suggestions.

File: _data/navigation.yml (hunk 3 above)
Comment (mrshirts) [reply _data/navigation.yml]: Looks like an issue with how the navigation processing scripts orders things - it put the software analysis before the other items, and it's unclear why.  Perhaps @ptmerz can take a quick look when he has a chance.

**Comment by davidlmobley** [comment]
Looks good to me, pending resolution of remaining comments eg to @ptmerz and @jppiquem .

**Pull request merged**